apps/odyssey-stats/src/app.scss

```diff
@@ -69,21 +69,10 @@
 		}
 	}
 
-	// Black segmented controls.
-	.is-section-stats ul.segmented-control li.segmented-control__item {
-		.segmented-control__link:hover {
-			background-color: var(--color-neutral-0);
-		}
-
-		&.is-selected .segmented-control__link,
-		&.is-selected .segmented-control__link:hover {
-			background-color: var(--studio-black);
-			border-color: var(--studio-black);
-		}
-
-		&.is-selected + .segmented-control__item .segmented-control__link {
-			border-left-color: var(--studio-black);
-		}
+	// Disable focus box-shadow from links.
+	// The box shadow is set to admin color which does not match the jetpack green.
+	a:focus {
+		box-shadow: none;
```

Comment:
The default box-shadow in WP-Admin is blue color, which does not look nice and does not match the studio-green color from Jetpack

![Arc -2025-06-04 at 15 19 08@2x](https://github.com/user-attachments/assets/4efa45cc-bb8d-4188-a30a-7f0279cdf921)

Reply:
Nice one!